.circleci/config.yml

```
@@ -23,7 +23,7 @@ jobs:
       - run:
           name: Install TeX
           command: |
-            sudo apt-get install texlive texlive-latex-extra latexmk
+            sudo apt-get install texlive texlive-latex-extra latexmk texlive-xetex texlive-fonts-extra libffcall-dev xindy
```

```suggestion
            sudo apt-get install texlive texlive-latex-extra latexmk texlive-xetex fonts-freefont-otf xindy
```
As of now, we only need the FreeSerif font, which is in this package. `latex-fonts-extra` is 1.4Gb so my vote would be to only add the fonts we need (and can expand in the future).

`libffcall-dev` is a dependency of `xindy`, so no need to specify it.